src/includes/getting-started-config/flutter.mdx

```diff
@@ -11,3 +11,11 @@ Future<void> main() async {
   // or define SENTRY_DSN via Dart environment variable (--dart-define)
 }
 ```
+
+<Note>
+
+Use the `SentryFlutter.init(...)` instead of `Sentry.init(...)` as it adds the Flutter integrations on top of the Dart SDK.
+
+</Note>
```

Comment:
I think we can remove this, our code snippet already uses `SentryFlutter` anyway

Reply:
oh, great. That content had been on the Advanced Usage page, so I moved it off of there, but better to remove.